hokusai/VERSION

```diff
@@ -1 +1 @@
-0.5.9
+0.5.10
```

Comment:
Would not update the version yet.  The normal workflow is to create a "prepare version x.x.x" commit / PR including updates to the Changelog once we are ready to release - see dfea0c7545486d52c9a08f27915d64bc7165631b

Reply:
Reverted VERSION.